ui/src/App.tsx

```diff
@@ -1,36 +1,213 @@
 import React from 'react';
-import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import clsx from 'clsx';
+import { css } from 'emotion';
+import { makeStyles } from '@material-ui/core/styles';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Drawer from '@material-ui/core/Drawer';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import List from '@material-ui/core/List';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import IconButton from '@material-ui/core/IconButton';
+import Badge from '@material-ui/core/Badge';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import MenuIcon from '@material-ui/icons/Menu';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import DashboardIcon from '@material-ui/icons/Dashboard';
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
 import { Home } from './screens/Home';
 import { Tests } from './screens/Tests';
-import { css } from 'emotion';
-import { Header } from './components/Header';
-import { GlobalStyles } from './utils/GlobalStyles';
+import { StylesProvider } from '@material-ui/core/styles';
+
+const drawerWidth = 240;
 
-const appStyles = css``;
+const useStyles = makeStyles((theme) => ({
+  root: {
```

Comment:
I've copied these from an example that didn't use JSS, but we could refactor this to use Emotion styles.